### PR TITLE
DOC: Add quick start

### PR DESCRIPTION
**Pull request by pllim**
**Description**

This PR adds a quick start section for lazy people like me. Would have helped when I don't care about running the pipeline but just wants to know quickly how to read your product.

Checklist
- [x] Tests -- N/A
- [x] Documentation
- [x] Change log -- N/A
- [ ] Milestone -- I cannot add this, so you have to
- [ ] Label(s) -- I cannot add this, so you have to. Do you know that Action can auto-add labels?

### CODE REVIEW

**Comment by pllim** [comment]
Example to load both to `astropy.nddata.CCDData` is even better, but when I tried with `astropy` 4.3.dev, `CCDData` does not understand GWCS; alas!

**Comment by pllim** [comment]
Rendered doc: https://jwst-pipeline--5976.org.readthedocs.build/en/5976/jwst/data_products/quickstart.html

File: docs/jwst/data_products/quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Now, you would do this instead:
+
+.. doctest-skip::
+
+    >>> from jwst import datamodels
+    >>> filename = 'myasdfinfits.fits'
+    >>> with datamodels.open(filename) as dm:
+    ...     mydata = dm.data  # Same as SCI
+    ...     mydata_unit_str = dm.bunit_data
```
Comment (pllim) [inline docs/jwst/data_products/quickstart.rst:29]: Didn't know about this until #5990 . I thought it was just `dm.bunit`!

**Comment by nden** [comment]
I am going to close this one because the documentation has been reworked.

**Pull request closed without merging**